From d90534469c5c43bf2a97e5698a5ddb4b7471f92a Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Tue, 24 Jul 2018 10:53:16 +0800
Subject: [PATCH] libldb: fix musl libc unkoown type error

tevent.h:1440:8: error: unknown type name 'pid_t'; did you mean 'div_t'?
        pid_t *pid,
         ^~~~~
         div_t

Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 lib/tevent/tevent.h | 2 ++
 1 file changed, 2 insertions(+)

--- a/lib/tevent/tevent.h
+++ b/lib/tevent/tevent.h
@@ -33,6 +33,7 @@
 #include <sys/time.h>
 #include <sys/types.h>
 #include <stdbool.h>
+#include <sys/stat.h>
 
 /* for old gcc releases that don't have the feature test macro __has_attribute */
 #ifndef __has_attribute
